src/services/verifiedCampaigns/verifiedCampaigns.service.js

```diff
@@ -10,47 +10,49 @@ module.exports = function verifiedCampaigns() {
 
   const service = {
     async create(data, params) {
-      const { txHash, image, slug } = data;
+      const { txHash, url, slug } = data;
 
       const projectInfo = await givethIoAdapter.getProjectInfoBySLug(slug);
-      const {
-        id: givethIoProjectId,
-        title,
-        description,
-        walletAddress: ownerAddress,
-      } = projectInfo;
-      if (params.user.address.toLowerCase() !== ownerAddress.toLowerCase()) {
+      const { id: givethIoProjectId, title, description, image } = projectInfo;
+      const owner = await givethIoAdapter.getUserByUserId(projectInfo.admin);
+      if (params.user.address.toLowerCase() !== owner.walletAddress.toLowerCase()) {
         throw new errors.Forbidden('The owner of project in givethIo is not you');
       }
       let campaign = await findCampaignByGivethIoProjectId(app, givethIoProjectId);
       if (campaign) {
         throw new errors.BadRequest('Campaign with this givethIo projectId exists');
       }
+      const pianataBaseUrl = 'https://gateway.pinata.cloud';
```

Comment:
Remove it, use below suggestion
```
            const imageIpfsPath = image.match(/\/ipfs\/.*/);
              picture: imageIpfsPath ? imageIpfsPath[0] : image,
```

Reply:
@aminlatifi Nice suggestion, thanks.
 done.